### DISCUSSION TITLE
不显示favicon怎么解决？

### DISCUSSION THREAD

**Discussion by lyizhuo** [spencerwooo/onedrive-vercel-index#142] [Q&A] [answered]
不显示favicon怎么解决？

**Comment by spencerwooo** [comment, marked as answer]
正常情况放到 `public` 文件夹下的 favicon 替换掉默认图标都能正常显示，如果不显示可能是你自己 favicon 的问题，可以尝试使用 [favicon.io](https://favicon.io/) 来正确导出格式匹配的 favicon.
